lib/os_allocation.c

```diff
@@ -1055,8 +1055,18 @@ verify_mba_id(const unsigned mba_id, const struct pqos_cpuinfo *cpu)
 	return ret;
 }
 
+/*
+ * @brief Set the bitmask for a given COS
+ *
+ * @param [in] l3cat_id L3 cat identifier
+ * @param [in] num_cos Total number of COS
+ * @param [in] ca Bitmask information to set
+ *
+ * @return Operations status
+ * @retval PQOS_RETVAL_OK on success
+ */
```

Comment:
Documentation should be moved to os_allocation.h header file

Reply:
ok. will do.